lib/interjectable.rb

```diff
@@ -5,12 +5,35 @@
 module Interjectable
   MethodAlreadyDefined = Class.new(StandardError)
 
+  INJECTED_METHODS = :injected_methods
+  INJECTED_METHODS_IVAR = :"@#{INJECTED_METHODS}"
```

Comment:
WDYT of just inlining these? Normally I am a big fan of constants, but since this module is included in many places, it means the constant itself will leak into many classes, etc. Would we need like `Module.injected_constants` to get around that? and so on.

Reply:
Good call! I'll update